src/topp/DatabaseSuitability.cpp

```diff
@@ -428,6 +439,23 @@ class DatabaseSuitability :
     }
     return true;
   }
+
+  bool scoreHigherThanFDR_(const PeptideHit& hit, double FDR, bool q_value_score)
```

Comment:
I would probably call it scoreBetterThanFDR_ since the score is actually lower.

Reply:
Yeah.. I think I changed the name but forgot to change the return values. Since I use this to check if a hit should be skipped, I thought `scoreHigherThanFDR_` would be the better name. But you are right: true and false should be the other way around.